scripts/write_deployinst_input.py

```diff
@@ -188,66 +187,124 @@ def read_template(template):
     return output_template
 
 
-def reactor_render(list, template, mox_template, output_file):
+def refine_name(name_data):
+    """ Takes the name data and decodes and refines it.
+
+    Parameters
+    ----------
+    name_data: str
+        reactor name data from csv file
+
+    Returns
+    -------
+    name: str
+        refined and decoded name of reactor
+    """
+
+    name = name_data.decode('utf-8')
+    start = name.find('(')
+    end = name.find(')')
+    if start != -1 and end != -1:
+        name = name[:start]
+    return name
+
+def reactor_render(data_list, output_file, is_cyborg=False):
     """Takes the list and template and writes a reactor file
 
     Parameters
-    ---------
-    list: list
+    ----------
+    data_list: list
         list of data on reactors
     template: jinja.template
         jinja template for reactor file
     mox_template: jinja.template
         jinja template for mox reactor file
     output_file: str
         name of output file
+    is_cyborg: bool
+        if True, uses Cyborg templates
 
     Returns
     -------
     The reactor section of cyclus input file
 
     """
 
-    for data in list:
-        # BWRs have different fuel assembly size, assembly per core and batch
-        name = data['reactor_name'].decode('utf-8').title()
-        start = name.find('(')
-        end = name.find(')')
-        if start != -1 and end != -1:
-            name = name[:start]
-        if data['type'].decode('utf-8') == 'BWR':
-            reactor_body = template.render(
+    template_path = '../templates/[reactor]_template_cyborg.xml.in'
+
+    if not is_cyborg:
+        template_path = template_path.replace('_cyborg', '')
+
+    pwr_template = read_template(template_path.replace('[reactor]', 'pwr'))
+    mox_reactor_template = read_template(template_path.replace('[reactor]', 'mox'))
+    candu_template = read_template(template_path.replace('[reactor]', 'candu'))
+
+    ap1000_spec = {'template' : pwr_template,
+                   'assem_size': 446.0,
+                   'n_assem_core': 157,
+                   'n_assem_batch': 52}
+    bwr_spec = {'template' : pwr_template,
+                   'assem_size': 180,
+                   'n_assem_core': 764 / 1000,
+                   'n_assem_batch': 764 / 3000}
+    phwr_spec = {'template' : candu_template,
+                   'assem_size': int(8000 / 47g3),
+                   'n_assem_core': 473 / 500,
+                   'n_assem_batch': 60}
+    candu_spec = {'template' : candu_template,
+                   'assem_size': int(8000 / 473),
+                   'n_assem_core': 473 / 500,
+                   'n_assem_batch': 60}
+    pwr_spec = {'template' : pwr_template,
+                   'assem_size': 446.0,
+                   'n_assem_core': 193 / 1000,
```

Comment:
if this needs to be an integer, then you need to say int(193/1000) because this integer/integer division will be a float in python 3 (although it will be an integer in python 2)